### PR TITLE
Spring boot remote simple example

### PR DESCRIPTION
**Pull request by karesti**
Should wait infinispan-spring-boot 2.0.0.Beta2

### CODE REVIEW

File: spring-boot/remote/src/main/java/org/infinispan/tutorial/simple/spring/remote/RemoteClientApp.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+   private RemoteCacheManager cacheManager;
+
+   public static void main(String[] args) {
+      SpringApplication.run(RemoteClientApp.class, args);
+   }
+
+   @Bean
+   public CommandLineRunner commandLineRunner(ApplicationContext ctx) {
+      return args -> {
+         RemoteCache<String, String> cache = cacheManager.getCache("default");
```
Comment (galderz) [inline spring-boot/remote/src/main/java/org/infinispan/tutorial/simple/spring/remote/RemoteClientApp.java]: Could the example also include how a second remote cache could also be `@AutoWired`? That way the example would show both `RemoteCacheManager` and `RemoteCache` auto wired. Thoughts?

Comment (karesti) [reply spring-boot/remote/src/main/java/org/infinispan/tutorial/simple/spring/remote/RemoteClientApp.java]: Why not! I will try that 💃

**Comment by karesti** [comment]
@galderz I released beta2 with spring boot 2, infinispan 9.2.3.Final and the corrections needed to inject remote caches

**Comment by karesti** [comment]
@galderz can be merged now. dependencies are in maven central 💃

**Pull request merged**